### ISSUE TITLE
Feature Request - left-to-right

### ISSUE THREAD

**Issue by Jay-PC** [shivarajnaidu/ng-marquee#6]
For many languages, people read right-to-left. With this marquee, the text scrolls right-to-left, which means they would have to read sentences backwards. How hard would it be to expose a simple parameter next to duration, something like "ltr=true"?

**Comment by yuvarajvemula** [comment]
Please try the updated version @Jay-PC

**Comment by Jay-PC** [comment]
We updated to your latest version and direction appears to be working (we tried left and right). However, the scroll speed is super fast and "duration" settings of any kind do not change it. We saw you changed "duration" to "speed" so we changed our code accordingly, but we pack our ng-marquee with many news headlines and the speed is super fast. Setting to the lowest speed of "drowsy" helps, but it would be better if we could control it with a number . Also, the mouse-over stop of the scroll is no longer working. We attempted that a few time on different browsers, restarting our angular app, and restarting firefox. Looking forward to the working version. Thanks for the upgrade--getting close to an awesome tool.
